### PR TITLE
fix spacing of bash scripts

### PR DESCRIPTION
**Pull request by jm96441n**
### Description

Fixed spacing of variable declarations, fixed shellcheck violations

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] appropriate backport labels added
* [ ] not a security concern

### CODE REVIEW

File: build-support/scripts/gen-changelog.sh [hunk 1]
```diff
@@ -66,8 +71,10 @@ esac
...
 
-read -ep $'Please enter the changelog message:\n' msg
+read -erp $'Please enter the changelog message:\n' msg
 
 echo -e "\`\`\`release-note:$type\n$msg\n\`\`\`" >>"$filename"
 
-cat .changelog/$pr_number.txt
+echo -e "\nChangelog added to $filename. Contents:\n"
+
+cat ".changelog/$pr_number.txt"
```
Comment (missylbytes) [inline build-support/scripts/gen-changelog.sh:80]: Nitpick, but this is broken for enterprise changelogs as their filenames start with "_".

Comment (missylbytes) [reply build-support/scripts/gen-changelog.sh:80]: I think should probably be $filename instead?

Comment (jm96441n) [reply build-support/scripts/gen-changelog.sh:80]: yup it should be!

Comment (jm96441n) [reply build-support/scripts/gen-changelog.sh]: done and done

**Review by missylbytes — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15,1.17,1.18,1.19] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15,1.17,1.18,1.19] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15,1.17,1.18,1.19] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @jm96441n! a backport is missing for this PR [21760] for versions [1.15] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Pull request merged**